src/Functions/Functions.Autorest/test/New-AzFunctionApp.ACA.Tests.ps1

```diff
@@ -0,0 +1,200 @@
+if(($null -eq $TestName) -or ($TestName -contains 'New-AzFunctionApp.ACA'))
```

Comment:
Please add the liscense above newly introduced files. You can refer to other files to copy and paste it here.